python/cugraph-pyg/cugraph_pyg/nn/conv/__init__.py

```diff
@@ -11,18 +11,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from .gat_conv import GATConv
-from .gatv2_conv import GATv2Conv
-from .hetero_gat_conv import HeteroGATConv
-from .rgcn_conv import RGCNConv
-from .sage_conv import SAGEConv
-from .transformer_conv import TransformerConv
+import warnings
 
-__all__ = [
-    "GATConv",
-    "GATv2Conv",
-    "HeteroGATConv",
-    "RGCNConv",
-    "SAGEConv",
-    "TransformerConv",
-]
+HAVE_CUGRAPH_OPS = False
+try:
+    import pylibcugraphops
+
+    HAVE_CUGRAPH_OPS = True
+except ImportError:
+    pass
+except Exception as e:
+    warnings.warn(f"Unexpected error while importing pylibcugraphops: {e}")
+
+if HAVE_CUGRAPH_OPS:
```

Comment:
Why is this PR putting cugraph-ops stuff behind a conditional import instead of completely removing it?

Or, said another way.... how are you expecting that someone would have `cugraph-pyg=25.2.*` and `pylibcugraphops=25.2.*` installed together? Is this being left here so it's still possible to build and install `pylibcugraphops` from source?

I'm hoping we can **completely stop publishing `pylibcugraphops` packages** in the 25.02 release.

Reply:
The conditional import is there because we planned to migrate cugraph-ops to a new location, and the new package will possibly retain the same name. We understand that rapids won't release cugraph-ops in 25.02. It's more of a placeholder for the migrated package.

Reply:
Oh interesting, did not know that!

Ok thank you.

Reply:
This entire file is now just being deleted: https://github.com/rapidsai/cugraph-gnn/pull/99#issuecomment-2587655322

Resolving this thread.